backend/corpora/common/entities/geneset.py

```diff
@@ -44,3 +47,31 @@ def create(cls, session, geneset_id: str, dataset_id: str):
         session.add(link)
         session.commit()
         return cls(link)
+
+    @classmethod
+    def get(cls, session, geneset_id: str, dataset_id: str):
+        try:
+            links = (
+                session.query(cls.table)
+                .filter(cls.table.geneset_id == geneset_id, cls.table.dataset_id == dataset_id)
+                .all()
+            )
+            return links[0]
+        except IndexError:
+            return None
+
+    @classmethod
+    def update_links_for_a_dataset(cls, session, dataset_id: str, add: list, remove: list):
```

Comment:
Added error check if add and remove are none.
```suggestion
    def update_links_for_a_dataset(cls, session, dataset_id: str, add: typing.Optional[list]=None, remove: typing.Optional[list]=None):
```